keyboards/ash1800/readme.md

```diff
@@ -0,0 +1,15 @@
+# ash1800
+
+![ash1800](https://i.imgur.com/YP173rP.jpg)
+
+ASH-1800 is an attempt to make a cheap g80/g81 clone.
+
+Keyboard Maintainer: [angelbirth](https://github.com/angelbirth)  
```

Comment:
extra spaces behind the next three lines or <br> otherwise formatting is messed up on Markdown. Thanks